autogen/build.rs

```diff
@@ -192,7 +192,7 @@ fn main() {
         instructions.ops,
     );
     write_formatted(
-        &autogen_src_dir.join("../rspirv/sr/autogen_module.rs"),
-        instructions.module_logic,
+        &autogen_src_dir.join("../rspirv/lift/autogen_context.rs"),
```

Comment:
Shouldn't we call this as `autogen_lift_context.rs` to be clear then?

Reply:
I had it called this for some time but then figured that "lift" is already in the path, so there is no need to repeat ourselves here. I don't mind either way :)
@AIOOB - do you have a preference?